evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -397,3 +399,25 @@ def test_arima_regressor_respects_use_covariates(mock_predict, mock_fit, ts_data
     assert "y" in mock_fit.call_args.kwargs
     mock_predict.assert_called_once()
     assert "X" not in mock_predict.call_args.kwargs
+
+
+@pytest.mark.parametrize("use_covariates", [True, False])
+def test_arima_regressor_can_forecast_arbitrary_dates(use_covariates, ts_data):
+    X, _, y = ts_data()
+
+    X_train, X_test, y_train, y_test = split_data(
+        X,
+        y,
+        problem_type="time series regression",
+        test_size=0.2,
+        random_seed=0,
+    )
+
+    X_test_last_5 = X_test.tail(5)
+
+    arima = ARIMARegressor(use_covariates=use_covariates)
+    arima.fit(X_train, y_train)
+
+    assert (
+        arima.predict(X_test).tail(5).tolist() == arima.predict(X_test_last_5).tolist()
+    )
```

Comment:
Does this fail when run on current main?

Reply:
yes!